src/hb-ot-layout.cc

```diff
@@ -250,6 +306,22 @@ hb_ot_layout_get_glyphs_in_class (hb_face_t                  *face,
   return face->table.GDEF->table->get_glyphs_in_class (klass, glyphs);
 }
 
+
+/**
+ * hb_ot_layout_get_attach_points:
+ * @face: The #hb_face_t to work on
+ * @glyph: The #hb_codepoint_t code point to query
+ * @start_offset: The offset into the attachment-point array to start returning
+ * @point_count: (in) The number of attachment points to return
+ *               (out) The number of attachment points returned in the array
+ * @point_array: (out) The array of attachment points found for the query
```

Comment:
Missing a colon after `(out)`.

Reply:
No problem, but (for my understanding) is that a style issue or does it affect building gtk-doc?

Reply:
It breaks gobject-introspection, and I *think* the HTML generated by GTK-Doc will also be different (these annotations take different styling).

Reply:
See how `[out]` is styled here https://harfbuzz.github.io/harfbuzz-hb-buffer.html#hb-buffer-get-glyph-infos.

Reply:
Are there any other annotations worth adding?

Reply:
Well, if there are, that's for later.

Reply:
I think `(array length=...)` and `(transfer none)` (as appropriate, search for them in existing files).

Reply:
Also `(allow-none)`.